tests/unit/test_scorer.py

```diff
@@ -22,11 +15,10 @@ def it_can_construct_randomscorer(self, request):
         score = random_scorer(tile)
 
         assert isinstance(random_scorer, scorer.RandomScorer)
-        assert isinstance(random_scorer, scorer.Scorer)
```

Comment:
Why are you deleting this? I think that is important for a scorer to be also a Scorer, i.e. implementing the interface

Reply:
RandomScorer is a Scorer by default considering that its signature is `RandomScorer(Scorer)` isn't it?